plugin.video.frikanalen/addon.py

```diff
@@ -27,55 +30,108 @@
 
 plugin = routing.Plugin()
 
+# fake gettext style translation using kodi approach with IDs
+def _(msg):
+    map = {
+        "Direkte":                    30001,
+        "Sendeplan":                  30002,
+        "Kategorier":                 30003,
+        "Søk":                        30004,
+        "Frikanalen akkurat nå (SD)": 30005,
+        "Frikanalen akkurat nå (HD)": 30006,
+    }
+    if msg in map:
+        return Addon().getLocalizedString(map[msg])
+    else:
+        return msg
+
+
 @plugin.route('/')
 def root():
     items = [
-        (plugin.url_for(live), ListItem("Direkte"), True),
-        (plugin.url_for(schedule), ListItem("Sendeplan"), True),
+        (plugin.url_for(live), ListItem(_("Direkte")), True),
+        (plugin.url_for(schedule), ListItem(_("Sendeplan")), True),
+        (plugin.url_for(categories), ListItem(_("Kategorier")), True),
+        (plugin.url_for(search), ListItem(_("Søk")), True),
     ]
     addDirectoryItems(plugin.handle, items)
     endOfDirectory(plugin.handle)
 
 @plugin.route('/live')
 def live():
-    addon_handle = plugin.handle
-
-    xbmcplugin.setContent(addon_handle, 'videos')
+    xbmcplugin.setContent(plugin.handle, 'videos')
 
-    li = ListItem('Frikanalen-sending (SD)', iconImage='DefaultVideo.png')
+    li = ListItem(_('Frikanalen akkurat nå (SD)'), iconImage='DefaultVideo.png')
     li.setProperty('IsPlayable', 'true')
```

Comment:
Iconimage in deprecated please use setart

Reply:
OK.  I'll change this and update the pull request branch.